shirow/request.py

```diff
@@ -61,6 +78,8 @@ def ret_and_continue(self, value):
         self._callback(self._get_successful_response(value, False))
 
     def ret_error(self, message):
+        """Causes a remote procedure to exit and inform the the client that an
+        error occurred.
+        """
         self._callback(self._get_error_response(message))
         raise Ret()
-
```

Comment:
Is it some need to remove the string?

Reply:
yeah, pylint complains when it sees so called `trailing newlines`.